databricks/koalas/internal.py

```diff
@@ -782,16 +782,34 @@ def pandas_df(self):
                                for name in index_names]
         return pdf
 
+    def with_new_sdf(self, sdf: spark.DataFrame,
+                     data_columns: Optional[List[str]] = None) -> '_InternalFrame':
+        """ Copy the immutable _InternalFrame with the updates by the specified Spark DataFrame.
+
+        :param sdf: the new Spark DataFrame
+        :param data_columns: the new column names.
+            If None, the original one is used.
+        :return: the copied _InternalFrame.
```

Comment:
Although i'm not sure if it is really necessary,

but how are you think about we have a simple example for these kinda util methods?
(since sometimes i got a little confused when i try to use such kinda of methods 😿 )

Reply:
I think these are simpler versions of `copy`.
I'm afraid I'm not quite sure where you are being confused.
Maybe you can add some examples (and description?) to make them clearer?

Reply:
@ueshin 

Ah, sorry my comment was totally not enough.

i meant how about we have a simple example for util methods or methods those are only used under-the-hood looks like the below. (it didn't necessarily talk about `with_new_sdf`)

```python
def combine_frames(this, *args, how="full"):
    """
    ...
    Examples
    --------
    >>> from databricks.koalas.config import set_option
    >>> set_option("compute.ops_on_diff_frames", True)
    >>> kdf = ks.DataFrame({'a': [1, 2, 3], 'b': [4, 5, 6]})
    >>> kser = ks.Series([10, 20, 30])
    >>> kdf
       a  b
    0  1  4
    1  2  5
    2  3  6
    >>> kser
    0    10
    1    20
    2    30
    Name: 0, dtype: int64
    
    >>> combine_frames(kdf, kser)
      this    that
         a  b    0
    0    1  4   10
    1    2  5   20
    2    3  6   30
    """
    ...
```

(if we add such examples for under-the-hood methods like the above, i think it's okay to just skip their doctest)